snooty.toml

```diff
@@ -13,15 +11,14 @@ product-source-repo = "https://github.com/mongodb/mongo-csharp-analyzer"
 product-home-page = "{+product-source-repo+}"
 product-nuget-link = "https://www.nuget.org/packages/MongoDB.Analyzer"
 product-package-name = "MongoDB.Analyzer"
-product-version = "1.2"
+product-version = "1.3"
 product-version-full = "{+product-version+}.0"
 product-settings-file-constant = "mongodb.analyzer.json"
 product-jira = "https://jira.mongodb.org/browse/VS"
 
 query-api = "MongoDB Query API"
 
-driver-long = "MongoDB .NET driver"
-driver-short = ".NET driver"
-driver-version = "2.19"
+driver-long = "MongoDB .NET/C# driver"
+driver-short = ".NET/C# driver"
```

Comment:
Question:
These seem like good changes to apply to all versions of the docs. Are these changes going to be backported? If so, it could be easier to create a separate PR for changes to backport so that the changes can be applied to each branch with minimal merge conflicts.

Reply:
good call. i'll make those changes after i do this ticket.